pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/EtterlatteBrev.kt

```diff
@@ -4,11 +4,16 @@ import no.nav.pensjon.brevbaker.api.model.Kroner
 import java.time.LocalDate
 
 data class BarnepensjonEtterbetaling(
-    val fraDato: LocalDate,
-    val tilDato: LocalDate,
-    val etterbetalingsperioder: List<BarnepensjonBeregningsperiode> = listOf()
+    val inneholderKrav: Boolean?,
```

Comment:
Vil disse tingene kun være aktuelt for BP og ikke OMS?

Reply:
I denne omgangen iallefall, men vil vel kanskje tro at det brevene for OMS skal endres litt etterhvert også.

Reply:
Men det med frivillig skattetrekk er vel mest relevant for BP tror jeg?

Reply:
Ja, det er det nok - ved OMS er det neppe frivillig